src/resources/images/alert.svg

```diff
@@ -1 +1,7 @@
-<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 18 18"><defs/><g fill="none" fill-rule="evenodd" transform="translate(1.533333 1.533333)"><circle cx="7.46666667" cy="7.46666667" r="7.46666667" stroke="#DA394D" stroke-linecap="round" stroke-linejoin="round" stroke-width="1.5"/><circle cx="7.46666667" cy="11.2" r="1.06666667" fill="#DA394D" fill-rule="nonzero"/><path stroke="#DA394D" stroke-linecap="round" stroke-linejoin="round" stroke-width="1.5" d="M7.46666667 3.2V8"/></g></svg>
\ No newline at end of file
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 18 18">
```

Comment:
do we need to SVGO (minify) this?

Reply:
I did run it through this site: https://jakearchibald.github.io/svgomg/

Reply:
weird, ok